modules/hcp-dc-hypervisor.adoc

```diff
@@ -0,0 +1,170 @@
+[id="hcp-dc-hypervisor_{context}"]
+= Configuring the hypervisor for a disconnected installation of {hcp}
+
+The following information applies to virtual machine environments only.
+
+.Procedure
+
+. To deploy a virtual management cluster, access the required packages by entering the following command:
++
+[source,terminal]
+----
+$ sudo dnf install dnsmasq radvd vim golang podman bind-utils net-tools httpd-tools tree htop strace tmux -y
+----
+
+. Enable and start the Podman service by entering the following command:
++
+[source,terminal]
+----
+$ systemctl enable --now podman
+----
+
+. To use `kcli` to deploy the management cluster and other virtual components, install and configure the hypervisor by entering the following commands:
++
+[source,terminal]
+----
+$ sudo yum -y install libvirt libvirt-daemon-driver-qemu qemu-kvm
+----
++
+[source,terminal]
+----
+$ sudo usermod -aG qemu,libvirt $(id -un)
+----
++
+[source,terminal]
+----
+$ sudo newgrp libvirt
+----
++
+[source,terminal]
+----
+$ sudo systemctl enable --now libvirtd
+----
++
+[source,terminal]
+----
+$ sudo dnf -y copr enable karmab/kcli
+----
++
+[source,terminal]
+----
+$ sudo dnf -y install kcli
+----
++
+[source,terminal]
+----
+$ sudo kcli create pool -p /var/lib/libvirt/images default
+----
++
+[source,terminal]
+----
+$ kcli create host kvm -H 127.0.0.1 local
+----
++
+[source,terminal]
+----
+$ sudo setfacl -m u:$(id -un):rwx /var/lib/libvirt/images
+----
++
+[source,terminal]
+----
+$ kcli create network  -c 192.168.122.0/24 default
+----
+
+. Enable the network manager dispatcher to ensure that virtual machines can resolve the required domains, routes, and registries. To enable the network manager dispatcher, in the `/etc/NetworkManager/dispatcher.d/` directory, create a script named `forcedns` that contains the following content, replacing values as necessary to match your environment:
++
+[source,bash]
+----
+#!/bin/bash
+
+export IP="192.168.126.1" <1>
+export BASE_RESOLV_CONF="/run/NetworkManager/resolv.conf"
+
+if ! [[ `grep -q "$IP" /etc/resolv.conf` ]]; then
+export TMP_FILE=$(mktemp /etc/forcedns_resolv.conf.XXXXXX)
+cp $BASE_RESOLV_CONF $TMP_FILE
+chmod --reference=$BASE_RESOLV_CONF $TMP_FILE
+sed -i -e "s/dns.base.domain.name//" -e "s/search /& dns.base.domain.name /" -e "0,/nameserver/s/nameserver/& $IP\n&/" $TMP_FILE <2>
+mv $TMP_FILE /etc/resolv.conf
+fi
+echo "ok"
+----
++
+<1> Modify the `IP` variable to point to the IP address of the hypervisor interface that hosts the {product-title} management cluster.
+<2> Replace `dns.base.domain.name` with the DNS base domain name.
+
+. After you create the file, add permissions by entering the following command:
++
+[source,terminal]
+----
+$ chmod 755 /etc/NetworkManager/dispatcher.d/forcedns
+----
+
+. Run the script and verify that the output returns `ok`.
+
+. Configure `ksushy` to simulate baseboard management controllers (BMCs) for the virtual machines. Enter the following commands:
```

Comment:
Per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command, the following commands may use instructions such as --

- Installing required packages
- Creating the `sushy-service` wit hthe port 9000
- Reloading daemon to apply the system changes
- Enabling the `ksushy` service